sh/tests/head-and-tail_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
```

Comment:
should we add a check to ensure the student solution is not a simple `echo`?

Maybe something like
```bash
if [[ $(cat <path-to-student-solution> | grep echo | wc -l) -ne 0 ]];
then
  echo "Bad solution";
fi;
```
Or, viceversa, check that the solution is using `curl`, `head` and `tail` keywords
Or both checks :)

Reply:
fixed